ytypes/util_schema.go

```diff
@@ -228,6 +216,21 @@ func childSchema(schema *yang.Entry, f reflect.StructField) (*yang.Entry, error)
 	return nil, nil
 }
 
+// findFirstNonChoiceOrCase recursively traverses the schema tree and populates
```

Comment:
I think you have two implementations of this even in this code?

Reply:
Done.